src/device/pf_pdport.c

```diff
@@ -984,6 +1002,35 @@ static int pf_pdport_write_data_adj (
    return ret;
 }
 
+static int pf_pdport_write_interface_adj (
+   pnet_t * net,
+   const pf_ar_t * p_ar,
+   const pf_iod_write_request_t * p_write_req,
+   const uint8_t * p_bytes,
+   uint16_t p_datalength,
+   pnet_result_t * p_result)
+{
+   uint16_t pos = 0;
+   pf_get_info_t get_info;
+   pf_multiple_interface_mode_t mode;
+
+   get_info.result = PF_PARSE_OK;
+   get_info.p_buf = p_bytes;
+   get_info.is_big_endian = true;
+   get_info.len = p_datalength;
+
+   pf_get_interface_adjust (&get_info, &pos, &mode);
+   net->if_ctl.mode = mode;
+
+   LOG_INFO (
+      PNET_LOG,
+      "PDPORT(%d): PD interface adjust. LLDP name of device mode: %s\n",
```

Comment:
` "PDPORT(%d): PD interface adjust. PLC is setting LLDP mode: %s\n"`

Reply:
Done